homeassistant/components/mqtt/update.py

```diff
@@ -148,6 +131,11 @@ def config_schema() -> vol.Schema:
 
     def _setup_from_config(self, config: ConfigType) -> None:
         """(Re)Setup the entity."""
+        self._attr_device_class = self._config.get(CONF_DEVICE_CLASS)
+        self._attr_release_summary = self._config.get(CONF_RELEASE_SUMMARY)
+        self._attr_release_url = self._config.get(CONF_RELEASE_URL)
+        self._attr_title = self._config.get(CONF_TITLE)
+        self._entity_picture = self._config.get(CONF_ENTITY_PICTURE)
```

Comment:
This is a change in behavior by allowing device class etc. to be changed, is it intentional?

Reply:
Good point. For the other platforms we do this by design. This should be a separate PR.

Reply:
Removed the changes from this PR and opened: #100957